tests/test_yaml.py

```diff
@@ -0,0 +1,93 @@
+import importlib
+import os
+from unittest import mock
+import pytest
+
+from rpmlb.yaml import Yaml
+import helper
+
+
+@pytest.fixture
+def valid_yaml():
+    """A Yaml object with a valid YAML file."""
+    return Yaml(helper.get_valid_recipe_file())
+
+
+def test_init_loads_file():
+    yaml = Yaml(helper.get_valid_recipe_file())
+    assert yaml
+
+
+def test_init_load_default_loader_when_failing_to_import_c_loader():
```

Comment:
Below logic is for a case of getting ImportError, and run `from yaml import Loader` in `rpmlb/yaml.py`.
But I could not implement it. So, I appreciate if you give me some hints to implement that. 

```
    def __init__(self, file_path):
        try:
            from yaml import CLoader as Loader
        except ImportError:
            from yaml import Loader <= This logic!
```